models/multiphase/d3q27_pf_velocity/Dynamics.c.Rt

```diff
@@ -1323,24 +1414,18 @@ CudaDeviceFunction void updateBoundary(){
 		case NODE_Wall:
 			BounceBack();
 			break;
-		case NODE_EVelocity:
-			EVelocity();
-			break;
-		case NODE_WVelocity:
-			WVelocity();
-			break;
-		case NODE_NVelocity:
-			NVelocity();
-			break;
-		case NODE_Velocity_Y_neg:
-			Velocity_Y_neg();
-			break;
-		case NODE_EPressure:
-			EPressure();
+        <?R
+        for (ii in 1:6){
+        ?>
+        case NODE_<?R C(my_velocity_boundaries[ii])?>:
+			<?R C(my_velocity_boundaries[ii])?>();
 			break;
-		case NODE_WPressure:
-			WPressure();
+		case NODE_<?R C(my_pressure_boundaries[ii])?>:
+			<?R C(my_pressure_boundaries[ii])?>();
 			break;
+		<?R
+        }
+        ?>
```

Comment:
Convective boundaries should be also handled here, right? Or at least a comment should be add, that this should be adjusted if certain convective boundaries are uncommented